src/Umbraco.Web.UI.Client/src/common/services/clipboard.service.js

```diff
@@ -75,6 +97,21 @@ function clipboardService(notificationsService, eventsService, localStorageServi
     
     var service = {};
     
+    /**
+    * @ngdoc method
+    * @name umbraco.services.clipboardService#registrerPropertyClearingResolver
+    * @methodOf umbraco.services.clipboardService
+    *
+    * @param {string} function A method executed for every property and inner properties copied.
+    *
+    * @description
+    * Executed for all properties including inner properties when performing a copy action.
+    */
+   service.registrerClearPropertyResolver = function(resolver) {
```

Comment:
Unfortunately this is a spelling mistake: `registrer`

Reply:
Fixed in this PR here - https://github.com/umbraco/Umbraco-CMS/pull/8218